src/util/KvAuth0.js

```diff
@@ -357,6 +357,7 @@ export const MockKvAuth0 = {
 	getKivaId: () => undefined,
 	getLastLogin: () => 0,
 	getMfaEnrollToken: () => Promise.resolve({}),
+	fakeAuthAllowed: () => undefined,
```

Comment:
@emuvente I think this is ok, though I suppose it could return `false` as well...

Reply:
Where `fakeAuthAllowed` is used currently, anything falsy would work, but I just changed it to `false` to be more explicit.